src/Actions/Inform/Credits.cs

```diff
@@ -46,18 +46,20 @@ public override void Execute(ActionInput actionInput)
             {
                 if (cachedContents == null || (parameters.Length > 0 && parameters[0].ToLower() == "reload"))
                 {
-                    StreamReader reader = new StreamReader("Files\\Credits.txt");
-                    StringBuilder stringBuilder = new StringBuilder();
-                    string s;
-                    while ((s = reader.ReadLine()) != null)
+                    using (StreamReader reader = new StreamReader("Files\\Credits.txt"))
```

Comment:
nit: An option if you like it too: Modern C# actually has some newer syntactical sugar which can scope a `using` to the current scope without all the extra indentation stuff.  Like:
```
using StreamReader reader = new StreamReader("Files\\Credits.txt");
StringBuilder stringBuilder = new StringBuilder();
...
```
Still takes care of disposal guarantee when we exit the scope, we just get to indent like a regular var.